src/Service/Backup/BackupBackgroundService.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Passwordless.Common.Models.Backup;
+using Passwordless.Service.Models;
+using Passwordless.Service.Storage.Ef;
+
+namespace Passwordless.Service.Backup;
+
+public class BackupBackgroundService : BackgroundService
+{
+    private readonly IServiceProvider _serviceProvider;
+    private readonly ILogger<BackupBackgroundService> _logger;
+
+    private readonly PeriodicTimer _timer = new(TimeSpan.FromMinutes(15));
+
+    public BackupBackgroundService(
+        IServiceProvider serviceProvider,
+        ILogger<BackupBackgroundService> logger)
+    {
+        _serviceProvider = serviceProvider;
+        _logger = logger;
+    }
+
+    protected async override Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
@jrmccannon That one was meant to run background jobs on fixed times during the day. This would just be a queue that takes the oldest one first to be processed.